operations/_scripts/generate/ansible/generate_bitops_config.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+echo "In ansible generate_bitops_config.sh"
+
+cli="cli:
+      main-playbook: playbook.yml"
+
+if [[ -n $ST2_CONF_FILE ]]; then
+cli="$cli
+      extra-vars: \"${ST2_CONF_FILE}\""
+fi
```

Comment:
The bitops.config generation to pass extra vars looks like a hack.

Is there anything from Ansible side? 

Here's how the vars are currently passed:
https://github.com/bitovi/github-actions-deploy-stackstorm/blob/08cb927d4268b18f95a065c9d970f12d4ee4c55e/operations/deployment/ansible/playbook.yml#L3-L4

There should be a better way!
Perhaps there's another solution around `ansible.cfg`, `var_files`, `include_vars` and so on

Some resources to explore:
- https://stackoverflow.com/a/53257820
- https://serverfault.com/questions/589734/ansible-can-i-use-vars-files-when-some-files-do-not-exist